thorlcr/activities/msort/thsortu.cpp

```diff
@@ -819,8 +822,11 @@ class CJoinHelper : implements IJoinHelper, public CSimpleInterface
                         }
                     }
                     while (state == JSonfail);
+                    matchStats.noteGroup((lhsProgressCount - 1) - startMatchLhsProgressCount, 0);
+                    startMatchLhsProgressCount = (lhsProgressCount - 1);
```

Comment:
took me a moment to validate that lhsProgressCount was 1 past end of lhs group so -1 made sense.
I might be tempted to code as : 
```
rowcount_t nextStartMatchLhsProgressCount = lhsProgressCount -1;
matchStats.noteGroup(nextStartMatchLhsProgressCount - startMatchLhsProgressCount, 0);
startMatchLhsProgressCount = nextStartMatchLhsProgressCount;
```

Reply:
I will make that change before merging